src/components/Movie.jsx

```diff
@@ -0,0 +1,90 @@
+import Lottie from "lottie-react";
+import { useEffect, useState } from "react";
+import { useParams, useNavigate } from "react-router-dom";
+import "./css/Movie.css";
+import animation from "../assets/animations/loading.json";
+import star from "/star.svg";
+import arrow from "/arrow.svg";
+
+export const Movie = () => {
+  const { id } = useParams();
+  const navigate = useNavigate();
+  const [loading, setLoading] = useState(true);
+  const [movie, setMovie] = useState();
+
+  const fetchApi = () => {
+    fetch(
+      `https://api.themoviedb.org/3/movie/${id}?api_key=e185d0927f85272fbd0fd2526ecf0657&language=en-US`
+    )
+      .then((response) => response.json())
+      .then((movie) => {
+        console.log("response:", movie);
+        if (movie.success === false) {
```

Comment:
i would consider moving this if statement to the previous then() since fetch fails the first then() will lead to an error and it won't proceed to the next then